.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     <<: *linter-image
     steps:
       - checkout
-      - run: golangci-lint run
+      - run: golangci-lint --verbose --concurrency 4 run
```

Comment:
shouldn't the concurrency flag go into `.golangci.yml`? And why verbose?

Reply:
Reducing the concurrency was an attempt to address the OOMkill issue on CI. I don't think running this on our machine will have an issue so I intentionally keep this out of global config file. Apparently this is not enough I will continue to experiment